contao/templates/rsce/rsce_text_list.html5

```diff
@@ -2,30 +2,34 @@
 
 <?php $this->block('content'); ?>
 
+<?php $previewMode = Contao\System::getContainer()->get('contao.security.token_checker')->isPreviewMode() ?>
```

Comment:
Maybe we can define global variables over the content blocks for readability.

Reply:
What would you think about something like that? Otherwise I agree, move it higher up.

```
<?php 
    $previewMode = Contao\System::getContainer()->get('contao.security.token_checker')->isPreviewMode()
    $this->extend('block_searchable'); 
?>

<?php $this->block('content'); ?>
```